service/worker/scheduler/workflow.go

```diff
@@ -1035,6 +1037,7 @@ func (s *scheduler) getListInfo(inWorkflowContext bool) *schedpb.ScheduleListInf
 		Paused:            s.Schedule.State.Paused,
 		RecentActions:     util.SliceTail(s.Info.RecentActions, s.tweakables.RecentActionCountForList),
 		FutureActionTimes: s.getFutureActionTimes(inWorkflowContext, s.tweakables.FutureActionCountForList),
+		RunningWorkflows:  s.Info.RunningWorkflows,
```

Comment:
unfortunately we can't change the result of getListInfo without using versioning, since updateMemoAndSearchAttributes compares the new value with the previous value to determine whether to call UpsertMemo: if this changes then we'll have new calls to UpsertMemo on replay which isn't allowed :disappointed:

Reply:
Oops, thanks for the catch. I'll put this behind a version flag.

Reply:
Though if I did it behind a server version flag, that doesn't seem like it'd help without also versioning the scheduler workflow.

Reply:
I mean this version mechanism: https://github.com/temporalio/temporal/blob/main/service/worker/scheduler/workflow.go#L1523